mathics/builtin/__init__.py

```diff
@@ -37,6 +38,42 @@
 )
 
 
+def contributing_builtin_var(module, name) -> Optional[type]:
```

Comment:
If the return is sometimes (or often) `getattr(module, name)` then the return type is a `str` not a Python `type`.

Reply:
In this case, what we want is a `type`, because is something that we want to instantiate afterward.

Reply:
I just checked, and you are correct - `type` is the most precise.